categories_map.py

```diff
@@ -849,11 +849,19 @@
     "Validation/Shashlik",
   ],
   "heterogeneous": [
-    "CUDADataFormats/Common",
     "CUDADataFormats/BeamSpot",
+    "CUDADataFormats/CaloCommon",
```

Comment:
if these are supposed to be used in reco CUDA, I think that a reco signature is needed as well.

Reply:
@fwyzard , any idea if these will be used by reco CUDA?

Reply:
yes

Reply:
can you please update the PR to add these under reconstruction too

Reply:
shall I also add under "heterogeneous" all packages that do or will contain any kind of CUDA code ?

Reply:
that is a good question, dependency of a category X should not be responsibility of that category X.

Reply:
Sorry, what I meant is: should heterogeneous (currently, only CUDA) code be signed by the `heterogeneous` category ?
If the answer is "no", then what about the data formats just added under `reconstucrtion` ?

Reply:
I have a vague recollection that the last time this was discussed (sorry no reference) the outcome was **not** to add `heterogeneous` on every package with CUDA code, but to be assigned by other L2s when necessary. I'm fine with moving all other `CUDADataFormats` than `CUDADataFormats/Common` and `CUDADataFormats/StdDictionaries` only under `reconstruction`.

Reply:
Let me add (post-merge) that after some further thought I think I contradicted myself in above (assign `heterogneous` on PRs on demand, but leave `CUDADataFormats` packages out from `heterogeneous`), and leaving all `CUDADataFormats` packages (also) under `heterogeneous` is more consistent.